fem/bilininteg_vectorfe.cpp

```diff
@@ -536,10 +746,10 @@ void VectorFEMassIntegrator::AssemblePA(const FiniteElementSpace &trial_fes,
 
    symmetric = MQ ? MQ->IsSymmetric() : true;
 
-   if (dim == 3 && ((trial_fetype == mfem::FiniteElement::CURL &&
-                     test_fetype == mfem::FiniteElement::DIV) ||
-                    (trial_fetype == mfem::FiniteElement::DIV &&
-                     test_fetype == mfem::FiniteElement::CURL)))
+   if ((trial_fetype == mfem::FiniteElement::CURL &&
+        test_fetype == mfem::FiniteElement::DIV) ||
+       (trial_fetype == mfem::FiniteElement::DIV &&
+        test_fetype == mfem::FiniteElement::CURL))
```

Comment:
Similar to the other previous comment, I think these would be simpler to read using additional boolean variables.